api/jest-setup.ts

```diff
@@ -0,0 +1,7 @@
+// Ensure that process.env values are defined when jest runs code, to match the behaviour
+// configured by serverless's "useDotenv" option (which also uses dotenv-expand).
+
+import * as dotenv from 'dotenv';
+import { expand } from 'dotenv-expand';
```

Comment:
What does expand do?

Reply:
It interprets parts of environment variables as bash parameters and expands them. So something like

`MYVAR=drink${FAVOURITE_DRINK}`

Might have a value of drinktea when the env var is actually used. Serverless framework uses the expand library if you tell it to pick up dotenv config. This was causing problems for me because I have a value with a dollar sign in it that was behaving differently in serverless to jest. In serverless, the dollar sign needed escaping, but in jest it didn't because it wasn't using expand until I told it to. So I added that to this setup file.